ci-operator/step-registry/telco-bastion/setup/telco-bastion-setup-commands.sh

```diff
@@ -14,24 +14,25 @@ cat << EOF > ~/inventory
 sshd.bastion-telco ansible_ssh_user=tester ansible_ssh_common_args="-o ConnectTimeout=5 -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null" ansible_ssh_private_key_file=/var/run/ssh-pass
 EOF
 
-# We will switch to kcli in the near future
 cat << EOF > ~/ocp-install.yml
 ---
-- name: Prepare to run ansible-ipi-install playbook
+- name: Grab and run kcli to install openshift cluster
   hosts: all
   tasks:
   - name: Clone repo
     git:
-      repo: https://github.com/openshift-kni/baremetal-deploy.git
-      dest: "~/baremetal-deploy"
+      repo: https://github.com/karamb/kcli-openshift4-baremetal.git
+      dest: "~/kcli-openshift4-baremetal"
       version: master
       force: yes
     retries: 5
+  - name: Remove old installer vm
+    shell: kcli delete vm --yes cnfdc5-installer
```

Comment:
why are we removing just the installed and not deleteing the plan ?
`kcli delete plan --yes upstream_ci`

Reply:
That's just what I got from @karmab for rerunning. @karmab should I add or switch to `kcli delete plan` ?

Reply:
you can do both things, but it s probably cleaner to delete the plan all together